app/lib/presentation/pages/welcome_page.dart

```diff
@@ -43,53 +42,38 @@ class _WelcomePageState extends State<WelcomePage> {
 
   @override
   Widget build(BuildContext context) {
-    return PageEnclosureMolecule(
```

Comment:
We would like to keep the PageEnclosureMolecule, it is good structure for the page title sub title and more.

If it does not run try setting expendChild to true